README.md

```diff
@@ -32,8 +32,8 @@ CloudOnce has evolved a lot since then, and on June 1st 2016 it was made open-so
 
 ## Trollpants Game Studio games
 All the games created by Trollpants Game Studio have been released as open source. They all use CloudOnce and have been updated to run in Unity 2017.3. Check them out for examples of how CloudOnce can be used.
-* [Trollpants repository](https://github.com/jizc/Trollpants)
-* [Trollpants WebGL builds](https://jizc.github.io/Trollpants)
+* [Trollpants repositories](https://github.com/orgs/Trollpants/repositories)
```

Comment:
I think I'd prefer this point to `https://github.com/Trollpants`, as that has a readme giving a better overview. The link text can be `Trollpants GitHub`.